api/src/main/java/au/org/aodn/nrmn/restapi/controller/ObservableItemController.java

```diff
@@ -50,9 +53,23 @@ public ObservableItemDto newObservableItem(@Valid @RequestBody ObservableItemDto
         ObservableItem persistedObservableItem = observableItemRepository.save(newObservableItem);
         return mapper.map(persistedObservableItem, ObservableItemDto.class);
     }
+    
+    @GetMapping("/observableItem/{id}")
+    public ObservableItemGetDto findOne(@PathVariable Integer id) {
+        ObservableItem observableItem = observableItemRepository.findById(id).orElseThrow(ResourceNotFoundException::new);
+        return mapper.map(observableItem, ObservableItemGetDto.class);
+    }
 
     private void validatePost(@RequestBody @Valid ObservableItemDto sitePostDto) {
 
+        List<ValidationError> errors = new ArrayList<ValidationError>();
+
+        if(sitePostDto.getObsItemTypeId() == null)
+            errors.add(new ValidationError(ObservableItemDto.class.getName(), "obsItemTypeId", null, "Observable Item Type is required"));
+
+        if(sitePostDto.getSpeciesEpithet() == null || sitePostDto.getSpeciesEpithet().isEmpty())
+            errors.add(new ValidationError(ObservableItemDto.class.getName(), "speciesEpithet", null, "Species Epithet is required."));
+        
```

Comment:
Validated is here to do that in more gracefull way

Reply:
Actually this should just be a javax validation - there's no need to complicate this with validated/turning validated into a response (yes I said complicate)

Reply:
Will use javax validation.